runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierPlugin.java

```diff
@@ -440,13 +439,18 @@ private void onChatMessage(ChatMessage event)
 					break;
 				}
 			}
+
 		}
 	}
 
 	@Override
 	protected void startUp() throws  Exception
 	{
-		String configMessagerList = config.customMessages();
-		customMessages = Arrays.asList(configMessagerList.split(DELIMITER_REGEX));
+		splitMessages(config.customMessages());
+	}
+
+	private void splitMessages(String customMessage)
+	{
+		customMessages = Splitter.on(',').splitToList(customMessage);
```

Comment:
save the Splitter.on as constant, also, keep regex (e.g `private static final COMMA_SPLITTER = Splitter.on(Pattern.compile("\\s*,\\s*"))`)